pkg/local_workflows/output_workflow.go

```diff
@@ -99,14 +99,16 @@ func outputWorkflowEntryPoint(invocation workflow.InvocationContext, input []wor
 	config := invocation.GetConfiguration()
 	debugLogger := invocation.GetEnhancedLogger()
 
+	// Handle findings models, if none found, continue with the rest
+	err := handleContentTypeFindingsModel(invocation, input, outputDestination)
+	if err != nil {
+		return output, err
+	}
+
 	for i := range input {
 		mimeType := input[i].GetContentType()
 
 		if strings.HasPrefix(mimeType, content_type.LOCAL_FINDING_MODEL) {
```

Comment:
Suggestion: @sandor-trombitas Let's remove the handled FindingsModel data from the input slice and then we can remove this part in the loop.

Reply:
done 👍